auditlog/models/rule.py

```diff
@@ -735,3 +735,9 @@ def _update_vals_list(self, vals_list):
                 if isinstance(fieldvalue, models.BaseModel) and not fieldvalue:
                     vals[fieldname] = False
         return vals_list
+
+    def _update_registry(self):
+        """Update the registry after a modification on automation rules."""
```

Comment:
automation?

Apart from that, isn't better to refer that this is mimicking what is shown in `base_automation`?

Reply:
copy paste error :sweat: